src/Components/DateInputForm/DateInputForm.tsx

```diff
@@ -0,0 +1,25 @@
+﻿import React from "react";
+import "./DateInputForm.scss"
+
+
+export function DateInputForm() {
+    return (
+        <div >
+            <form className="form">
+                <label>DAY: 
+                    <br/>
+                    <input type="text" name="day"/>
+                </label>
+                <label>MONTH:
+                    <br/>
+                    <input type="text" name="month"/>
+                </label>
+                <label>YEAR:
+                    <br/>
+                    <input type="text" name="year"/>
+                </label>
```

Comment:
have you seen https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date ?

Should make your life a bit easier and should also be easier for the user.

Reply:
Changed to date input form with submit for the "Curiosity: on this date" page, intended to be used with rover image api in future.